From 232c5b6924652f65ef5c047754e84b87b5fc6ac8 Mon Sep 17 00:00:00 2001
From: Peter Korsgaard <peter@korsgaard.com>
Date: Sat, 3 Oct 2015 19:01:38 +0200
Subject: [PATCH] sunxi_x_g2d: drop unused dri2 include

The driver doesn't use DRI for anything.

[Upstream-status: submitted (https://github.com/ssvb/xf86-video-fbturbo/pull/45)]
Signed-off-by: Peter Korsgaard <peter@korsgaard.com>
---
 src/sunxi_x_g2d.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/sunxi_x_g2d.c b/src/sunxi_x_g2d.c
index d826027..8888abe 100644
--- a/src/sunxi_x_g2d.c
+++ b/src/sunxi_x_g2d.c
@@ -31,7 +31,6 @@
 #include "xf86_OSproc.h"
 #include "xf86.h"
 #include "xf86drm.h"
-#include "dri2.h"
 #include "damage.h"
 #include "fb.h"
 #include "gcstruct.h"
-- 
2.1.4

